### PR TITLE
[stable/jaeger] Fix helm warnings

### PR DESCRIPTION
**Pull request by karlhumma**
Signed-off-by: Karl Humma <karl.humma@qlik.com>

#### What this PR does / why we need it:
The purpose of this PR is to fix a value which causes helm to show a warning.
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #
```
2019/10/22 17:32:44 Warning: Building values map for chart 'jaeger-operator'. Skipped value (map[]) for 'resources', as it is not a table.
2019/10/22 17:32:44 Warning: Building values map for chart 'jaeger-operator'. Skipped value (map[]) for 'resources', as it is not a table.
```
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/jaeger-operator/values.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name:
 
-resources:
+resources: []
```
Comment (batazor) [inline stable/jaeger-operator/values.yaml]: maybe worth using: `resources: {}` :question:

**Comment by karlhumma** [comment]
ugh i messed that up. need to redo this one.

**Pull request closed without merging**